comp/trace/config/component.go

```diff
@@ -32,9 +32,12 @@ type Component interface {
 	// Warnings returns config warnings collected during setup.
 	Warnings() *config.Warnings
 
-	// SetHandler sets http handler for config
+	// SetHandler returns a handler for runtime configuration changes.
 	SetHandler() http.Handler
 
+	// GetCoreConfigHandler returns a handler to fetch the runtime configuration.
+	GetCoreConfigHandler() http.Handler
```

Comment:
Just curious about the meaning of `Core` in this context?

Reply:
I renamed it.